modules/040-control-plane-manager/openapi/doc-ru-config-values.yaml

```diff
@@ -3,6 +3,21 @@ properties:
     description: |
       Параметры `kube-apiserver`.
     properties:
+      admissionPlugins:
+        description: |
+          Включить дополнительные [admission plugin'ы](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers).
```

Comment:
Could you please synchronize English and Russian docs?

Reply:
Done.